Makefile

```diff
@@ -22,8 +23,19 @@ vm-sanity-checks:
 platform-sanity-checks:
 	cd platform_sanity_checks && ${MAKE} ${PASS_DOWN_ARGS}
 
+rmsr/rmsr.ko:
+ifeq ($(shell uname -s),Linux)
+	# Allow git to fail if rmsr has already been cloned.
+	${GIT} clone https://github.com/softdevteam/rmsr.git | true
```

Comment:
OK, so heretic thought: let's integrate the rsmr repo into this one? I mean, it could be a subdir, right?